bower.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "elemental",
+  "version": "0.0.0",
+  "description": "React UI Framework",
+  "main": "elemental.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elementalui/elemental-dist.git"
+  },
+  "dependencies": {
+    "react": "^0.14"
```

Comment:
There is a problem with bower dependencies. `react-dom` and `react-addons-css-transition-group` are not published in bower registry, so I did not add them here.
It is probably possible to set those deps directly to github